static/js/echo.js

```diff
@@ -93,6 +93,11 @@ function insert_local_message(message_request, local_id) {
     return message.local_id;
 }
 
+exports.is_slash_command = function (content) {
+    return !content.startsWith('/me') && content.startsWith('/');
```

Comment:
I think we want to make this smarter, checking against a list of valid slash commands in the organization.  Ideally, we'd send it down to the frontend in `zerver/lib/events.py` (ending up in `page_params`), and have the list live in the backend.

Feel free to just add this as a refactor on top.

Reply:
I'll push back on this, just because it's rare that somebody prefixes a message with slash, and the outcome of not doing local echo isn't really a bad thing, so I'm not sure we want the maintenance headache of sending down a list of available commands.  Thoughts?

Reply:
We'll want the list of commands for typeahead later, probably, but you're right that maybe that'll need some extra metadata anyway.  This is certainly fine for now.